src/libstd/sys/unix/process/process_common.rs

```diff
@@ -301,6 +314,7 @@ impl Stdio {
                 Ok((ChildStdio::Owned(theirs.into_fd()), Some(ours)))
             }
 
+            #[cfg(not(target_os = "fuchsia"))]
```

Comment:
I'm not in love with these inline cfgs, but they seem to match the way `redox` was treated above, and I don't think this would be more cleanly expressed via a separate file. :woman_shrugging:

Reply:
Agreed.